lightstep/sdk/metric/internal/syncstate/sync.go

```diff
@@ -183,20 +181,43 @@ type record struct {
 	// supports checking for no updates during a round.
 	collectedCount int64
 
-	// accumulator can be a multi-accumulator if there
-	// are multiple behaviors or multiple readers, but
-	// these distinctions are not relevant for synchronous
-	// instruments.
-	accumulator viewstate.Accumulator
-
-	// attributeSet is ordered and deduplicated
-	attributeSet attribute.Set
-
-	// attributeList is in user-specified order, may contain duplicates.
-	attributeList []attribute.KeyValue
+	// inst allows referring to performance settings.
+	inst *Observer
```

Comment:
question: is there a reason this is a pointer to the observer and not the performance setting itself?

Reply:
Not really a reason, but I was thinking ahead to adding new configuration, so the struct might grow and using a pointer to _something_ would be appropriate, maybe via an interface. I chose to point to the thing corresponding with a single instrument since I could imagine configuring per-instrument behavior and also per-view behavior. Ugh, already requested here: https://github.com/open-telemetry/opentelemetry-specification/pull/2960